test/unit/test_simd.cpp

```diff
@@ -1889,9 +1889,11 @@ typedef ::testing::Types<
                   simd_t<     simd<int,    0, simd_abi::sve>, int,    4>,
                   simd_t<simd_mask<double, 0, simd_abi::sve>, bool,   4>>,
 
+#if ARB_SVE_WIDTH >= 8
```

Comment:
So, if we cap the SVE length at 8 in the tests, I'd assume we should do so in all of our SIMD. Or we shouldn't cap it, ever. 
What's the reasoning here?